content/latest/deploy/kubernetes/helm-configuration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Helm Configuration
+linkTitle: Helm Configuration
+description: Helm Configuration
+aliases:
+  - /deploy/kubernetes/helm-configuration/
+menu:
+  latest:
+    identifier: helm-configuration
+    parent: deploy-kubernetes
+    weight: 621
+isTocNested: true
+showAsideToc: true
+---
+
+## Configure Cluster
+
+Instead of using the default values in the helm chart, you can also modify the configuration of the YugaByte cluster according to your requirements. The following section shows the commands and tags that can be modified to achieve the desired configuration.
+
+### CPU, Memory & Replica Count
+
+The default values for the Helm chart are in the `helm/yugabyte/values.yaml` file. The most important ones are listed below. As noted in the Prerequisites section above, the defaults are set for a 3 nodes Kubernetes cluster each with 4 CPU cores and 15 GB RAM.
+
+```
+persistentVolume:
+  count: 2
+  storage: 10Gi
+  storageClass: standard
+
+resource:
+  master:
+    requests:
+      cpu: 2
+      memory: 7.5Gi
+  tserver:
+    requests:
+      cpu: 2
+      memory: 7.5Gi
+
+replicas:
+  master: 3
+  tserver: 3
+
+partition:
+  master: 3
+  tserver: 3
+```
+
+If you want to change the defaults, you can use the command below. You can even do `helm install` instead of `helm upgrade` when you are installing on a Kubernetes cluster with configuration different than the defaults.
+
+```sh
+$ helm upgrade --set resource.tserver.requests.cpu=8,resource.tserver.requests.memory=15Gi yb-demo ./yugabyte
+```
+
+Replica count can be changed using the command below. Note only the tservers need to be scaled in a Replication Factor 3 cluster which keeps the masters count at 3.
+
+```sh
+$ helm upgrade --set replicas.tserver=5 yb-demo ./yugabyte
+```
+
+### LoadBalancer for Services
+
+By default, the YugaByte DB helm chart exposes only the master ui endpoint via LoadBalancer. If you wish to expose also the ycql and yedis services via LoadBalancer for your app to use, you could do that in couple of different ways.
+
+
+If you want individual LoadBalancer endpoint for each of the services (YCQL, YEDIS), run the following command.
+
+```sh
+$ helm install yugabyte -f expose-all.yaml --namespace yb-demo --name yb-demo --wait
+```
+
+If you want to create a shared LoadBalancer endpoint for all the services (YCQL, YEDIS), run the following command.
+
+```sh
+$ helm install yugabyte -f expose-all-shared.yaml --namespace yb-demo --name yb-demo --wait
+```
+
+You can also bring up an internal LoadBalancer (for either the masters' or the tservers' service) if required. Just specify the [annotation](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer) required for your cloud provider. The following command brings up an internal LoadBalancer for the tserver service(s) in GCP and AWS respectively.
+
+```sh
```

Comment:
@schoudhury - would be good to have this under the button style layout. Don't remember the widget for it though.

Reply:
Updated the PR with the in-place switch buttons.